core/tauri/src/manager/webview.rs

```diff
@@ -610,6 +610,19 @@ impl<R: Runtime> WebviewManager<R> {
         .expect("failed to run on_webview_created hook");
     }
 
+    if let Ok(webview_labels_array) = serde_json::to_string(&webview.manager.webview.labels()) {
+      let _ = webview.manager.webview.eval_script_all(format!(
+      "window.__TAURI_INTERNALS__.metadata.webviews = {webview_labels_array}.map(function (label) {{ return {{ label: label }} }})",
+    ));
+    }
+
+    let _ = webview.manager.emit(
+      "tauri://webview-created",
+      Some(crate::webview::CreatedEvent {
+        label: webview.label().into(),
+      }),
+    );
```

Comment:
why are we re-emitting same events here as the one in `WebviewBuilder::build`?

Reply:
ahh i may not have tested multi webview windows, can't remember. We can probably remove the event in WebviewBuilder::build ~~(or move this one here into the if let some in WindowBuilder::build_internal)~~

Reply:
I see, then just remove the one in WebviewBuilder and keep this one instead